frontend/src/pages/projects/ProjectDetailsContext.tsx

```diff
@@ -71,13 +83,17 @@ const ProjectDetailsContextProvider: React.FC = () => {
     useInferenceServices(namespace),
   );
   const serverSecrets = useContextResourceData<SecretKind>(useServingRuntimeSecrets(namespace));
+  const projectSharingRB = useContextResourceData<RoleBindingKind>(useProjectSharing(namespace));
+  const users = useContextResourceData<UserKind>(useUsers());
```

Comment:
Again, this is for typeahead, we might not want this if performance wise is a drawback

Reply:
Nope, don't do it in the Context -- definitely should be done in the tab. No user information is needed outside of the permissions tab.

Reply:
Ok, that's that, following the conversation above, I think we cannot filter by words, so it's either having this in it's own context or remove it.

Reply:
Ok, we can go with option one, ill reply in that part